pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -308,7 +420,7 @@ def load_files(files, tag=None, sat_id=None, altitude_bin=None):
         Series of filenames
     tag : str or NoneType
         tag or None (default=None)
-    sat_id : str or NoneType
+    inst_id : str or NoneType
         satellite id or None (default=None)
     altitude_bin : integer
```

Comment:
```suggestion
    altitude_bin : int
```

Same goes for `int`.